### PR TITLE
Fix mispelled path and package name

### PR DESCRIPTION
**Pull request by mcascardi**
Fix for issue #251 

There was a misspelled path that was causing gulp not to find the assets it was referencing.

### CODE REVIEW

**Comment by mcascardi** [comment]
@holger1411 can I get a review/merge on this?

**Comment by Thomas-A-Reinert** [comment]
TL:DR: I can´t help. Sorry. I wish I could.

**Long version:** 
Hey @mcascardi  - sorry, I can´t help. holger sold this repo to "@understrapframework" about one year ago if I recall correctly.
If I was able to merge anything, I´d done so. And I would´ve also quality-assured every merge request.

There have not been any merges for bout three months now and I wonder that the owner just showed up today and merged 10 merge requests within 3 mins. That also shows how much he cares about quality assurance, code reviews etc.

My recommendation: Fork this project and merge any updates into it as long as understraps future is kinda .. well, how shall I say? I think you get the picture anyway.
